plugins/amazonq/shared/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonq/lsp/AmazonQLanguageClientImpl.kt

```diff
@@ -425,6 +429,60 @@ class AmazonQLanguageClientImpl(private val project: Project) : AmazonQLanguageC
         return CompletableFuture.completedFuture(Unit)
     }
 
+    override fun sendPinnedContext(params: LSPAny): CompletableFuture<Unit> {
+        val chatManager = ChatCommunicationManager.getInstance(project)
+
+        // Get the active text editor and create text document identifier
+        val editor = FileEditorManager.getInstance(project).selectedTextEditor
+        val textDocument = editor?.let {
+            TextDocumentIdentifier(it.virtualFile.path)
```

Comment:
We are sending the URI in chat prompt, maybe we can follow the same implementation here, re: the comment above